src/Enemy/EnemyCap.cpp

```diff
@@ -0,0 +1,181 @@
+#include "Enemy/EnemyCap.h"
+#include <al/Library/Base/String.h>
+#include <al/Library/LiveActor/ActorActionFunction.h>
+#include <al/Library/LiveActor/ActorFlagFunction.h>
+#include <al/Library/LiveActor/ActorInitFunction.h>
+#include <al/Library/LiveActor/ActorModelFunction.h>
+#include <al/Library/LiveActor/ActorMovementFunction.h>
+#include <al/Library/LiveActor/ActorResourceFunction.h>
+#include <al/Library/LiveActor/LiveActorFunction.h>
+#include <al/Library/Math/MathAngleUtil.h>
+#include <al/Library/Movement/EnemyStateBlowDown.h>
+#include <al/Library/Movement/EnemyStateBlowDownParam.h>
+#include <al/Library/Nerve/NerveSetupUtil.h>
+#include <al/Library/Nerve/NerveUtil.h>
+#include <al/Library/Obj/PartsFunction.h>
+#include <al/Library/Placement/PlacementFunction.h>
+#include <al/Library/Yaml/ByamlUtil.h>
+#include "prim/seadSafeString.h"
+
+namespace {
+NERVE_IMPL(EnemyCap, Wait);
+NERVE_IMPL(EnemyCap, BlowDown);
+
+struct {
+    NERVE_MAKE(EnemyCap, Wait);
+    NERVE_MAKE(EnemyCap, BlowDown);
+} NrvEnemyCap;
+
+}  // namespace
+
+EnemyCap* EnemyCap::createEnemyCap(const char* name) {
+    return new EnemyCap(name);
+}
+
+EnemyCap::EnemyCap(const char* name) : al::LiveActor(name) {}
+
+static al::EnemyStateBlowDownParam fallbackStateBlowDownParam{};
+
+// NON_MATCHING: Unsure how the al::StringTmp here works
+void EnemyCap::initPartsFixFile(al::LiveActor* actor, const al::ActorInitInfo& initInfo, const char* archiveName, const char* suffix) {
+    mCap = actor;
+    mCapBaseMtx = mCap->getBaseMtx();
+    al::initChildActorWithArchiveNameNoPlacementInfo(this, initInfo, archiveName, 0);
+    al::initNerve(this, &NrvEnemyCap.Wait, 1);
+    mStateBlowDown = new al::EnemyStateBlowDown(this, &fallbackStateBlowDownParam, "吹き飛び状態");
+    al::initNerveState(this, mStateBlowDown, &NrvEnemyCap.BlowDown, "吹き飛び");
+
+    al::StringTmp<128> test;
+    al::createFileNameBySuffix(&test, "InitPartsFixInfo", suffix);
+    if (al::isExistModelResourceYaml(mCap, test.cstr(), 0)) {
+        al::ByamlIter resourceYaml (al::getModelResourceYaml(mCap, test.cstr(), 0));
+        const char* jointName = nullptr;
+        resourceYaml.tryGetStringByKey(&jointName, "JointName");
+        if (jointName)
+            mCapBaseMtx = al::getJointMtxPtr(mCap, jointName);
+        al::tryGetByamlV3f(&mLocalTrans, resourceYaml, "LocalTrans");
+        al::tryGetByamlV3f(&mLocalRotate, resourceYaml, "LocalRotate");
+        al::tryGetByamlV3f(&mLocalScale, resourceYaml, "LocalScale");
+        if (!al::isNearZero(mLocalTrans, 0.001f) || !al::isNearZero(mLocalRotate, 0.001f))
+            mIsAtOrigin = true;
```

Comment:
Seems to be the other way around - if *not* at origin and rotated correctly, this variable is set to `true`.